app/MMB/src/scenes/timeline/TimelineEventDateLocation.js

```diff
@@ -2,23 +2,51 @@
 
 import * as React from 'react';
 import { View } from 'react-native';
-import { StyleSheet, Text } from '@kiwicom/mobile-shared';
+import { StyleSheet, Text, Color } from '@kiwicom/mobile-shared';
 import { Translation } from '@kiwicom/mobile-localization';
 
+export const TimelineEventDateLocationContext = React.createContext({
+  highlightText: false,
+});
```

Comment:
Can we create contexts in a separate files? If I wanted to find this context comes from, this is not the first place I would look.

Reply:
I remedied to that and used the aptly named `context` folder :)